demisto_sdk/commands/common/constants.py

```diff
@@ -905,7 +907,12 @@ class GithubContentConfig:
     CONTENT_GITHUB_LINK: str
     CONTENT_GITHUB_MASTER_LINK: str
 
+    IS_GITLAB = False
+    GITLAB_ID: Optional[int] = None
+    BASE_RAW_GITLAB_LINK = "https://code.pan.run/api/v4/projects/{GITLAB_ID}/repository"
```

Comment:
@tomneeman151293 That's a good point. Since this URL is meant for inner usage I think it'll be best to init it to None, and allow to configure it via an ENV variable.
This will also be a more scalable solution.

Reply:
Should we add the organization as ENV variable as well?
Currently I'm assuming the gitlab url is `code.pan.run/xsoar/{some_repo}`.
Should the user define the URL and the organization and I will use it as `{URL}/{org}/some_repo`.

Reply:
@DeanArbel @tomneeman151293 Great thinking and a solution.
Let the user define the GITURL as whole.
wdyt?

Reply:
@ilaner missed your message, sorry
Gitlab URL is not code.pan.run
This is relevant only for PANW.
I agree that it is better to define as whole (if we see it annoys someone, we can think of a smarter way to define it. but currently the assumption of code.pan.run/xsoar in my opinion might not be 100% accurate)

Reply:
I changed it to use gitparser library on python, instead of using regex. The whole url is deducted, and I don't assume the url.